### PR TITLE
store/tikv: add metrics.

### PR DESCRIPTION
**Pull request by disksing**
@zimulala @coocood @shenli

### CODE REVIEW

File: store/tikv/client.go [hunk 1]
```diff
@@ -54,14 +54,18 @@ type rpcClient struct {
...
-		p: NewPools(maxConnecion, func(addr string) (*Conn, error) {
+		p: NewPools(maxConnection, func(addr string) (*Conn, error) {
 			return NewConnection(addr, dialTimeout)
 		}),
 	}
 }
 
 // SendCopReq sends a Request to co-processor and receives Response.
 func (c *rpcClient) SendCopReq(addr string, req *coprocessor.Request, timeout time.Duration) (*coprocessor.Response, error) {
+	sendReqCounter.WithLabelValues("cop").Inc()
```
Comment (shenli) [inline store/tikv/client.go]: Define a const is better.

File: store/tikv/client.go [hunk 2]
```diff
@@ -85,6 +89,10 @@ func (c *rpcClient) SendCopReq(addr string, req *coprocessor.Request, timeout ti
 
 // SendKVReq sends a Request to kv server and receives Response.
 func (c *rpcClient) SendKVReq(addr string, req *kvrpcpb.Request, timeout time.Duration) (*kvrpcpb.Response, error) {
+	sendReqCounter.WithLabelValues("kv").Inc()
```
Comment (shenli) [inline store/tikv/client.go]: ditto

File: store/tikv/coprocessor.go [hunk 3]
```diff
@@ -269,6 +273,7 @@ func buildCopTasks(bo *Backoffer, cache *RegionCache, ranges *copRanges, desc bo
 	if elapsed := time.Since(start); elapsed > time.Millisecond*500 {
 		log.Warnf("buildCopTasks takes too much time (%v), range len %v, task len %v", elapsed, rangesLen, len(tasks))
```
Comment (shenli) [inline store/tikv/coprocessor.go]: Task length is a useful info.

File: store/tikv/client.go (hunk 1 above)
Comment (siddontang) [inline store/tikv/client.go]: I think ok for metrics.

Comment (siddontang) [inline store/tikv/client.go]: I think it is ok for metrics.

File: store/tikv/coprocessor.go (hunk 3 above)
Comment (disksing) [inline store/tikv/coprocessor.go]: @shenli  Do you suggest to add a gauge for it?

Comment (shenli) [inline store/tikv/coprocessor.go]: Use a histogram is better.

Comment (disksing) [inline store/tikv/coprocessor.go]: @shenli Commit Addressed.

**Comment by shenli** [comment]
LGTM

File: server/conn.go [hunk 4]
```diff
@@ -534,7 +534,7 @@ func (cc *clientConn) handleQuery(sql string) (err error) {
 	startTs := time.Now()
```
Comment (zimulala) [inline server/conn.go]: s/startTs/startTS

**Comment by shenli** [comment]
@zimulala @tiancaiamao PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**